nucypher/utilities/prometheus/metrics.py

```diff
@@ -129,6 +129,11 @@ def start_prometheus_exporter(
     from twisted.web.resource import Resource
     from twisted.web.server import Site
 
+    # Disabling default collector metrics
+    REGISTRY.unregister(GC_COLLECTOR)
+    REGISTRY.unregister(PLATFORM_COLLECTOR)
+    REGISTRY.unregister(PROCESS_COLLECTOR)
```

Comment:
Process collector (RAM & CPU usage, etc.) sounds generally useful to me. How did we decide that this is irrelevant to our goals?

Reply:
Fair question.

I'm focused on having client metrics (I mean, Ursula metrics) to have enough information to properly calculate Threshold staking rewards. At the same time, I would like to make a clean-up in the current metrics removing all unuseful or redundant information to have only relevant information as tBTCv2 nodes do:

```
# TYPE btc_connectivity gauge
btc_connectivity 1 1694426887268

client_info{version="v2.0.0-m4"} 1

# TYPE connected_bootstrap_count gauge
connected_bootstrap_count 7 1694426947164

# TYPE connected_peers_count gauge
connected_peers_count 140 1694426947166

# TYPE eth_connectivity gauge
eth_connectivity 1 1694426887162

# TYPE tbtc_pre_params_count gauge
tbtc_pre_params_count 1000 1694426948350
```

In any case, this PROCESS_COLLECTOR has not been collecting information, so no information about CPU or RAM is exposed at all. So I would prefer to disable this collector which is not working anyway.

That said, I could agree that process information (RAM or CPU) may be useful for troubleshoothing or to better understand the performance of the nodes. But IMHO we should address this later on.

Reply:
I see how this metric is not relevant to the dashboard we're building and I agree it doesn't seem like a priority. Thanks for explaining!